spring-graalvm-native-configuration/src/main/java/org/springframework/boot/autoconfigure/data/RepositoriesHints.java

```diff
@@ -20,34 +20,43 @@
 import org.springframework.boot.autoconfigure.data.cassandra.CassandraRepositoriesAutoConfiguration;
 import org.springframework.boot.autoconfigure.data.couchbase.CouchbaseReactiveRepositoriesAutoConfiguration;
 import org.springframework.boot.autoconfigure.data.couchbase.CouchbaseRepositoriesAutoConfiguration;
+import org.springframework.boot.autoconfigure.data.elasticsearch.ElasticsearchRepositoriesAutoConfiguration;
+import org.springframework.boot.autoconfigure.data.elasticsearch.ReactiveElasticsearchRepositoriesAutoConfiguration;
 import org.springframework.boot.autoconfigure.data.mongo.MongoReactiveRepositoriesAutoConfiguration;
 import org.springframework.boot.autoconfigure.data.mongo.MongoRepositoriesAutoConfiguration;
 import org.springframework.core.io.support.PropertiesLoaderSupport;
 import org.springframework.graalvm.extension.NativeImageConfiguration;
 import org.springframework.graalvm.extension.NativeImageHint;
 import org.springframework.graalvm.extension.TypeInfo;
 
-@NativeImageHint(trigger = CassandraReactiveRepositoriesAutoConfiguration.class, typeInfos= {
-		@TypeInfo(types= {ConditionalOnRepositoryType.class, OnRepositoryTypeCondition.class,RepositoryType.class})
+@NativeImageHint(trigger = CassandraReactiveRepositoriesAutoConfiguration.class, typeInfos = {
+		@TypeInfo(types = {ConditionalOnRepositoryType.class, OnRepositoryTypeCondition.class, RepositoryType.class})
 })
-@NativeImageHint(trigger = CassandraRepositoriesAutoConfiguration.class, typeInfos= {
-		@TypeInfo(types= {ConditionalOnRepositoryType.class, OnRepositoryTypeCondition.class,RepositoryType.class})
+@NativeImageHint(trigger = CassandraRepositoriesAutoConfiguration.class, typeInfos = {
+		@TypeInfo(types = {ConditionalOnRepositoryType.class, OnRepositoryTypeCondition.class, RepositoryType.class})
```

Comment:
I know this is more of a reformatting of what is there than a real change but it caught my eye that I think the feature should be auto adding at least the first two of these, we should try testing with the first removed and then the second. (same applies across all these hints)

Reply:
thanks @aclement - checked without the hints and everything worked. Will undo the changes and open another PR that takes care of removing hints for the other store modules.